package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "swagger-nodegen-cli",
```

Comment:
To publish to the depot, the package `name` and `publishConfig` need to be update.  Refer to the JS SDK for an example.